modules/serverless-install-serving-web-console.adoc

```diff
@@ -51,4 +61,43 @@ image::serving-conditions-true.png[Conditions]
 ====
 It may take a few seconds for the Knative Serving resources to be created. You can check their status in the *Resources* tab.
 ====
-. If the conditions have a status of *Unknown* or *False*, wait a few moments and then check again after you have confirmed that the resources have been created.
++
+If the conditions have a status of *Unknown* or *False*, wait a few moments and then check again after you have confirmed that the resources have been created.
+
+. Check that the Knative Serving resources have been created:
++
+[source,terminal]
+----
+$ oc get pods -n knative-serving
+----
++
+.Example output
+[source,terminal]
+----
+NAME                               READY   STATUS    RESTARTS   AGE
+activator-5c596cf8d6-5l86c         1/1     Running   0          9m37s
+activator-5c596cf8d6-gkn5k         1/1     Running   0          9m22s
+autoscaler-5854f586f6-gj597        1/1     Running   0          9m36s
+autoscaler-hpa-78665569b8-qmlmn    1/1     Running   0          9m26s
+autoscaler-hpa-78665569b8-tqwvw    1/1     Running   0          9m26s
+controller-7fd5655f49-9gxz5        1/1     Running   0          9m32s
+controller-7fd5655f49-pncv5        1/1     Running   0          9m14s
+webhook-5c7d878c7c-n267j           1/1     Running   0          9m35s
```

Comment:
This output is a little bit old? :thinking: Currently DomainMapping should be deployed by default.
Here is the recent output if you want to replace.

```
NAME                                                        READY   STATUS      RESTARTS   AGE
activator-67ddf8c9d7-p7rm5                                  2/2     Running     0          4m
activator-67ddf8c9d7-q84fz                                  2/2     Running     0          4m
autoscaler-5d87bc6dbf-6nqc6                                 2/2     Running     0          3m59s
autoscaler-5d87bc6dbf-h64rl                                 2/2     Running     0          3m59s
autoscaler-hpa-77f85f5cc4-lrts7                             2/2     Running     0          3m57s
autoscaler-hpa-77f85f5cc4-zx7hl                             2/2     Running     0          3m56s
controller-5cfc7cb8db-nlccl                                 2/2     Running     0          3m50s
controller-5cfc7cb8db-rmv7r                                 2/2     Running     0          3m18s
domain-mapping-86d84bb6b4-r746m                             2/2     Running     0          3m58s
domain-mapping-86d84bb6b4-v7nh8                             2/2     Running     0          3m58s
domainmapping-webhook-769d679d45-bkcnj                      2/2     Running     0          3m58s
domainmapping-webhook-769d679d45-fff68                      2/2     Running     0          3m58s
storage-version-migration-serving-serving-0.26.0--1-6qlkb   0/1     Completed   0          3m56s
webhook-5fb774f8d8-6bqrt                                    2/2     Running     0          3m57s
webhook-5fb774f8d8-b8lt5                                    2/2     Running     0          3m57s
```

Reply:
Thank you so much!